packages/fxa-auth-server/test/local/senders/renderer.ts

```diff
@@ -53,7 +54,7 @@ describe('Renderer', () => {
 
     it('handles escaped quote format', () => {
       const { key, val } = splitPlainTextLine(
-        `${pair.key}="${pair.val} \"baz\" "`
+        `${pair.key}="${pair.val} "baz" "`
```

Comment:
Is this testing the same thing anymore? Might want to put an eslint ignore next line here?

Reply:
The tests before/after are equivalent to one-another.

Reply:
I understand that the escape is not needed in the template string, but I'm referring to what the test is trying to check.

IIUC, the test is checking to see that even if escape characters are used in the email text, that the method correctly splits the key from the value when we're trying to localize the text. If we remove those escape chars from the input, then this is no longer testing what it intended to test, right?

Reply:
Those escape characters aren't even present in the string, then. If you wanted the escape character to be present in the string, they'd need to be `\\`.

Removing the test would not be part of this scope of work imo.

Reply:
I just updated the test to use `\\` in local, and the test fails. This may be a real bug if that must be supported. We'll want to look into this later.

Reply:
Thanks for looking into it a bit more -- can you file a follow-up for this and link this comment?